### PR TITLE
[DOCUMENTATION] Add return types whenever possible to promises

### PR DESCRIPTION
**Pull request by Serabe**
Addresses #15114 in emberjs/ember.js

### CODE REVIEW

File: packages/ember-testing/lib/helpers/wait.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ import { pendingRequests } from '../test/pending_requests';
 
   @method wait
   @param {Object} value The value to be returned.
-  @return {RSVP.Promise}
+  @return {RSVP.Promise<Object>} Promise that resolves to the passed value.
```
Comment (stefanpenner) [inline packages/ember-testing/lib/helpers/wait.js]: isn't this `<any>` ?

**Review by stefanpenner — CHANGES_REQUESTED** [review]

File: packages/ember-testing/lib/helpers/wait.js (hunk 1 above)
Comment (Serabe) [inline packages/ember-testing/lib/helpers/wait.js]: I used object because it is the type specified for `value`, changing it.

**Comment by Serabe** [comment]
I've added the changes. Sorry for the delay, got too many things on my plate :$

**Review by locks — APPROVED** [review]

**Comment by rwjblue** [comment]
Thank you @Serabe!

**Pull request merged**